test/expect.php

```diff
@@ -0,0 +1,11 @@
+<?hh
+
+namespace Facebook\HackCodegen;
+
+function expect<T>(T $obj, mixed ...$args): CodegenExpectObj<T> {
+  return new CodegenExpectObj(new ImmVector(\func_get_args()));
+}
+
+function expect_with_context<T>(string $context, T $obj, mixed ...$args): CodegenExpectObj<T> {
+  return new CodegenExpectObj(new ImmVector(\array_slice(\func_get_args(), 1)), $context);
```

Comment:
ditto

Reply:
The base class (in another repo) still uses collections (https://github.com/hhvm/fbexpect/blob/master/src/ExpectObj.php)